src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -52,9 +54,18 @@
   }
 
   private final boolean useCompactMapEntries;
+  private final BehaviorOnNullValues behaviorOnNullValues;
 
-  public DataConverter(boolean useCompactMapEntries) {
+  /**
+   * @param behaviorOnNullValues How to treat records with null values. Cannot be null; if in doubt,
+   * {@link BehaviorOnNullValues#DEFAULT} can be used.
```

Comment:
If you add JavaDoc to any method, that JavaDoc should be complete: it should have a good description, `@param` for every parameter, `@return` if the method has a return, and any thrown exceptions. In this case, you're missing the `@param useCompactMapEntries`.

When you build, are you using `mvn clean install`? I'm surprised that checkstyle didn't catch this.

Reply:
I am using `mvn clean install` to build--should checkstyle be refusing to build for me when encountering errors like this, or do I have to look through the build output for warnings and/or through the reports directory for results files?